react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,53 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+  It is a a development server that uses Webpack to compile React, JSX, and ES6, auto-prefix CSS files.
```

Comment:
This looks to be more of a description of Create-React-App rather than React itself. I would think of React as a library for creating frontend UIs.